### PR TITLE
optimize imports, fixed build with cython 0.23.1

### PR DESCRIPTION
**Pull request by bgaifullin**

### CODE REVIEW

File: src/xmlsec/enc.pyx [hunk 1]
```diff
@@ -47,6 +44,7 @@ cdef class EncryptionContext:
             raise InternalError("failed to create encryption context")
 
         self._handle = handle
+        print("!!!", XMLSEC_ENC_RETURN_REPLACED_NODE)
```
Comment (mehcode) [inline src/xmlsec/enc.pyx]: :fire:

Comment (bgaifullin) [inline src/xmlsec/enc.pyx]: sorry, I was forgot to remove this.

**Comment by mhahn** [comment]
This fixed the previous issue I was seeing, but now I'm getting the following error running this on ubuntu 14.04:
```
# pip install git+https://github.com/bgaifullin/python-xmlsec.git
You are using pip version 7.1.0, however version 7.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting git+https://github.com/bgaifullin/python-xmlsec.git
  Cloning https://github.com/bgaifullin/python-xmlsec.git to /tmp/pip-TzF4GC-build
Requirement already satisfied (use --upgrade to upgrade): lxml>=3.0 in /usr/local/lib/python2.7/dist-packages (from xmlsec==0.3.1)
Installing collected packages: xmlsec
  Running setup.py install for xmlsec
    Complete output from command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-TzF4GC-build/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-prP32C-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-2.7
    creating build/lib.linux-x86_64-2.7/xmlsec
    copying src/xmlsec/meta.py -> build/lib.linux-x86_64-2.7/xmlsec
    copying src/xmlsec/__init__.py -> build/lib.linux-x86_64-2.7/xmlsec
    copying src/xmlsec/error.py -> build/lib.linux-x86_64-2.7/xmlsec
    running build_ext
    cythoning src/xmlsec/constants.pyx to src/xmlsec/constants.c
    building 'xmlsec.constants' extension
    creating build/temp.linux-x86_64-2.7
    creating build/temp.linux-x86_64-2.7/src
    creating build/temp.linux-x86_64-2.7/src/xmlsec
    x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -DXMLSEC_NO_CRYPTO_DYNAMIC_LOADING=1 -DXMLSEC_OPENSSL_098=1 -DXMLSEC_NO_GOST=1 -D__XMLSEC_FUNCTION__=__FUNCTION__ -DXMLSEC_CRYPTO_OPENSSL=1 -DXMLSEC_NO_XKMS=1 -DXMLSEC_NO_SIZE_T= -DXMLSEC_CRYPTO=openssl -Isrc -I/usr/local/lib/python2.7/dist-packages/lxml/includes -I/usr/local/lib/python2.7/dist-packages/lxml -I/usr/include/xmlsec1 -I/usr/include/libxml2 -I/usr/include/python2.7 -c src/xmlsec/constants.c -o build/temp.linux-x86_64-2.7/src/xmlsec/constants.o -DXMLSEC_CRYPTO_OPENSSL=1
    src/xmlsec/constants.c: In function 'initconstants':
    src/xmlsec/constants.c:2419:61: error: 'xmlSecTransformEcdsaSha1Id' undeclared (first use in this function)
       __pyx_t_2 = ((PyObject *)__pyx_f_6xmlsec_9constants__mkti(xmlSecTransformEcdsaSha1Id)); if (unlikely(!__pyx_t_2)) {__pyx_filename = __pyx_f[0]; __pyx_lineno = 104; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
                                                                 ^
    src/xmlsec/constants.c:2419:61: note: each undeclared identifier is reported only once for each function it appears in
    src/xmlsec/constants.c:2431:61: error: 'xmlSecTransformEcdsaSha224Id' undeclared (first use in this function)
       __pyx_t_2 = ((PyObject *)__pyx_f_6xmlsec_9constants__mkti(xmlSecTransformEcdsaSha224Id)); if (unlikely(!__pyx_t_2)) {__pyx_filename = __pyx_f[0]; __pyx_lineno = 105; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
                                                                 ^
    src/xmlsec/constants.c:2443:61: error: 'xmlSecTransformEcdsaSha256Id' undeclared (first use in this function)
       __pyx_t_2 = ((PyObject *)__pyx_f_6xmlsec_9constants__mkti(xmlSecTransformEcdsaSha256Id)); if (unlikely(!__pyx_t_2)) {__pyx_filename = __pyx_f[0]; __pyx_lineno = 106; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
                                                                 ^
    src/xmlsec/constants.c:2455:61: error: 'xmlSecTransformEcdsaSha384Id' undeclared (first use in this function)
       __pyx_t_2 = ((PyObject *)__pyx_f_6xmlsec_9constants__mkti(xmlSecTransformEcdsaSha384Id)); if (unlikely(!__pyx_t_2)) {__pyx_filename = __pyx_f[0]; __pyx_lineno = 107; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
                                                                 ^
    src/xmlsec/constants.c:2467:61: error: 'xmlSecTransformEcdsaSha512Id' undeclared (first use in this function)
       __pyx_t_2 = ((PyObject *)__pyx_f_6xmlsec_9constants__mkti(xmlSecTransformEcdsaSha512Id)); if (unlikely(!__pyx_t_2)) {__pyx_filename = __pyx_f[0]; __pyx_lineno = 108; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
                                                                 ^
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

    ----------------------------------------
Command "/usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-TzF4GC-build/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-prP32C-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-TzF4GC-build```

**Comment by bgaifullin** [comment]
the cython compiler older than 0.23 does not  automatically include corresponding .pxd file.
in new version it generates same code in both cases: when manually included corresponding .pxd file and when not.

**Pull request merged**